.github/workflows/README.md

```diff
@@ -0,0 +1,35 @@
+# Github actions README
+
+This is an introduction about auto-cherry-pick workflow.
+
+
+take 202205 branch for example:
+1. pr_cherrypick_prestep:
+```mermaid
+graph
+Start(Origin PR) --> A{merged?}
+A -- NO --> STOP
+A -- YES --> A1{Approved<br> for 202205<br> Branch?}
+A1 -- NO --> STOP
+A1 -- YES --> A2(pr_cherrypick_prestep)
+B(pr_cherrypick_prestep)
+B -->  B1{cherry pick<br>conflict?}
+B1 -- YES --> B2(Add tag:<br>Cherry Pick Confclit_202205) --> B3(Add comment:<br>refer author code conflict) --> STOP1(STOP)
+B1 -- NO --> B4(Create New PR) --> B5(New PR add tag:<br> automerge) --> B6(New PR add comment:<br>Origin PR link) --> B7(Origin PR add tag:<br>Created PR to 202205 Branch) --> B8(Origin PR add comment:<br>New PR link)
```

Comment:
We need to add a step after "B1 -- YES " to check whether the PR has already created, if yes, we do not need to create a new PR for it.